relay-spans/src/span.rs

```diff
@@ -169,7 +169,7 @@ pub fn otel_to_sentry_span(otel_span: OtelSpan) -> EventSpan {
                 "http.route" | "url.path" => {
                     http_route = otel_value_to_string(value);
                 }
-                key if key.contains("exclusive_time_ns") => {
+                key if key.contains("exclusive_time_nano") => {
```

Comment:
Do they actually suffix stuff with `_nano` and not `_nanos`?

Reply:
Yes, for example `StartTimeUnixNano` https://opentelemetry.io/docs/specs/otel/metrics/data-model/.